ksqldb-cli/src/test/java/io/confluent/ksql/cli/console/ConsoleTest.java

```diff
@@ -1094,4 +1091,19 @@ public void shouldThrowOnInvalidCliPropertyValue() {
     assertThat(terminal.getOutputString(),
         containsString("Invalid value BURRITO for configuration WRAP: String must be one of: ON, OFF, null"));
   }
+
+  @Test
+  public void shouldPrintAssertTopicResult() {
+    // Given:
+    final KsqlEntityList entities = new KsqlEntityList(ImmutableList.of(
+        new AssertTopicEntity("statement", "topic")
```

Comment:
As an optional thing, it may be worth considering a new other than "topic" for the topic.  That avoids the approved version being "Topic topic exists."

*shrugs*

Reply:
lol okay. Done.